shell-pipeline.yml

```diff
@@ -14,6 +14,10 @@ jobs:
       BOSH_CLIENT: {{development-bosh-client}}
       BOSH_CLIENT_SECRET: {{development-bosh-client-secret}}
       BOSH_CA_CERT: master-bosh-root-cert/master-bosh.crt
+      CREDHUB_CA_CERT: master-bosh-root-cert/master-bosh.crt
+      CREDHUB_CLIENT: ((developmentbosh/credhub-admin-client))
```

Comment:
I don't think this is going to work. At the moment, concourse namespaces credential lookups by team and by pipeline. So this will actually to resolve to the path `/concourse/main/jumpbox/developmentbosh/credhub-admin-client`, which isn't going to exist. Also, for this to work, we'd need to have a single credhub that stores credentials for all deployments, or to copy credentials across separate credhubs, which could be confusing.

Reply:
ugh, yeah. 😞 

I like the approach of having it pull from manifests, but then it's not quite going to work with pipelines, unless we pull that manifest into this pipeline just for this. thoughts?

Reply:
I think I'm starting to butt my head up against some of these problems as well.  I'm attempting to get `developmentbosh` to generate some credentials for the alb-domain-broker.  At the very least, to run these deployments from concourse, concourse needs to have the ability to use development's credhub (at least, that's my assertion).  It seems though that we need a strategy like so: https://github.com/pivotal-cf/pcf-pipelines/blob/master/docs/credhub-integration.md#-integration-topologies.

Reply:
Also, for what it's worth, I'm starting to be of the opinion that it would be a _lot_ easier to deploy concourse along with the various deployments (dev, staging, etc.) a al BUCC, at least from the standpoint of maintainability.